runtime/frequency/src/apis.rs

```diff
@@ -152,12 +149,14 @@ impl_runtime_apis! {
 		}
 
 		fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
-			get_preset::<RuntimeGenesisConfig>(id,  &genesis_config_presets::get_preset)
+			get_preset::<RuntimeGenesisConfig>(id,  &crate::genesis::presets::get_preset)
 		}
 
 		fn preset_names() -> Vec<sp_genesis_builder::PresetId> {
 			vec![
 				sp_genesis_builder::PresetId::from("development"),
+				sp_genesis_builder::PresetId::from("paseo"),
+				sp_genesis_builder::PresetId::from("frequency"),
```

Comment:
For `frequency` and `frequency-testnet` can just return the parsed JSON spec files.

For `frequency-no-relay` and `frequency-local` the generated ones are good.